flink-runtime-web/web-dashboard/src/app/pages/submit/submit.component.ts

```diff
@@ -23,7 +23,7 @@ import { Router } from '@angular/router';
 import { JarFilesItemInterface } from 'interfaces';
```

Comment:
should we rename the `JarFilesItemInterface` to `ArtifactFilesItemInterface` in `jar.js`?